### ISSUE TITLE
Add llms.txt

### ISSUE THREAD

**Issue by darinkishore** [jlowin/fastmcp#61]
It's important to have an LLMs.txt to enable LLMs to work with FastMCP easier!  [llms.txt](https://llmstxt.org/)

Right now, the readme more than suffices, but it would be good to have an LLMs.txt that's automatically updated to match the readme (or vice versa) with special formatting to make it easier on LM eyes.

**Comment by ajram23** [comment]
Can you please add support for this. Would be very valuable!

**Comment by jlowin** [comment]
Good call, we should add it to the repo! I'll set up.an action to keep it up to date. 

In the meantime, the documentation has both [gofastmcp.com/llms.txt](https://gofastmcp.com/llms.txt) and [gofastmcp.com/llms-full.txt](https://gofastmcp.com/llms-full.txt) which doesn't include the source code but is quite complete.

**Comment by ajram23** [comment]
Thanks @jlowin! That was quick! much appreciated!

**Comment by jlowin** [comment]
On further review, it appears that the standard is to add this for documentation and not for source code, in which case it is already satisfied!
